app/src/org/commcare/activities/CommCareHomeActivity.java

```diff
@@ -158,10 +164,24 @@
 
     private boolean sessionNavigationProceedingAfterOnResume;
 
+    //REMINDER STUFF
+    private CommCareReminderService mBoundService;
+    private ServiceConnection mConnection;
+    private Object serviceLock = new Object();
+    boolean mIsBound = false;
+    boolean mIsBinding = false;
+    private Vector<Alert> allAlerts = new Vector<>();
+
     @Override
     protected void onCreateSessionSafe(Bundle savedInstanceState) {
         super.onCreateSessionSafe(savedInstanceState);
 
+        for(Suite s: CommCareApplication._().getCurrentApp().getCommCarePlatform().getInstalledSuites()){
+            allAlerts.addAll(s.getAlerts());
+        }
+
+        launchService(allAlerts);
```

Comment:
I think we want to avoid launching this particular service unless we did get alerts out of the suite. IE: For most apps it would be best if the new service didn't launch at all.